### DISCUSSION TITLE
Add ability to group fields into tabs

### DISCUSSION THREAD

**Discussion by jordie23** [directus/directus#4083] [Feature Requests]
I suggest a new interface option to "Presentation & Aliases", similar to divider but would display the fields as tabs.

Perhaps the first "tab" field added in would designate where the tabs would appear. So for example, if you wanted a few tabs but then a title field to be above them all, you could do:

```
Title Field
Tab1
Field1
Field2
Tab2
Field3
Field4
```

In the above case, the `Title Field` would always be visible, and you'd have a Tab navigation below that field where you can choose Tab1 and Tab2 which divides the rest of the fields.

You could also ensure all fields are inside a tab by having a tab be the first field:

```
Tab1
Title Field
Field1
Field2
Tab2
Field3
Field4
```

With this, the tab navigation would be the first thing below the "Editing Item in Reviews" and the save buttons. The Title Field would appear inside Tab1.

Doing it this way would allow for more flexibility than I've seen on other CMS's. 

I know you've got a huge roadmap, so I don't expect this any time soon, but figured I'd pop it here on the list as I couldn't seem to see it here already when searching.

**Comment by benhaynes** [comment]
Love it! The main question is should we do this as one-off logic, or wait until we have field grouping figured out and implemented. 🤔

**Reply by rijkvanzanten** [reply]
Yeah this is definitely a part of the bigger Grouping discussion 🙂 https://github.com/directus/directus/discussions/3526

A tab-UI is nothing more than a single group that toggles between it's children fields. If one/more of the nested fields themselves are a group, you end up with the structure the OP mentioned:

```
Title Field (Regular Field)

Tabs (Group "parent")

  - Tab 1 (Nested Group)
    Field1
    Field2

  - Field3 (Second "tab" could be just 1 field)

  - Tab 2 (Nested Group)
    Field4
    Field5

**Reply by jordie23** [reply]
Sounds awesome! :)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by hanneskuettner** [comment]
You can install my extension and use this interface right now! :smile: https://github.com/hanneskuettner/directus-extension-group-tabs-interface
